src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -105,3 +113,24 @@ func CreateRemoveMsg(name string) sarama.ProducerMessage {
 		Headers: headers,
 	}
 }
+
+func AddFlagType(command *cobra.Command, coprocType *string) {
+	command.Flags().StringVar(
+		coprocType,
+		"type",
+		"nodejs",
+		"Coproc type engine. Can be: nodejs(default), data-policy",
+	)
+}
+
+func CheckCoprocType(coprocType string) error {
+	if coprocType == "nodejs" {
+		return nil
+	}
+
+	if coprocType == "data-policy" {
```

Comment:
Why we name coproc type `data-policy` shouldn't that be `v8` or `internal` or `embedded` ?

Reply:
We will have 3 type 
- nodejs
- datapolicy (it is for fetch integration without materialized topic)
- inline (it will looks like nodejs but with v8 stuff)